Tests/SpeziSchedulerTests/SchedulerTests.swift

```diff
@@ -286,18 +286,49 @@ final class SchedulerTests: XCTestCase {
         let taskId = UUID()
         let scheduledDate = Date()
         let scheduledEvent = Event(taskId: taskId, scheduledAt: scheduledDate)
-        
+
         let completedEvent = Event(taskId: taskId, scheduledAt: scheduledDate)
         completedEvent.complete(true)
-        
+
         var scheduledEventHasher = Hasher()
         scheduledEvent.hash(into: &scheduledEventHasher)
         let scheduledEventHash = scheduledEventHasher.finalize()
-        
+
         var completedEventHasher = Hasher()
         completedEvent.hash(into: &completedEventHasher)
         let completedEventHash = completedEventHasher.finalize()
-        
+
         XCTAssertEqual(scheduledEventHash, completedEventHash)
     }
+    func testTaskSchedulingWithStartDateAndEndDate() async throws {
```

Comment:
Thank you for adding some tests here. As we already work on these in https://github.com/StanfordSpezi/SpeziScheduler/pull/41 it might make sense to rather only include it there and include a test there that validates that the notification styling functionality is called correctly and that the content is used to then schedule a notification.